### PR TITLE
Updating Schedule, Fixing Schedule Scaling given event count

### PR DESCRIPTION
**Pull request by krishnakpandian**
Problem
=======
Schedule needs to be updated again given the changes on 1/13/22/

Solution
========
What I/we did to solve this problem
* Updated Schedule Props based on Execs information
* Changed Schedule SCSS to have component have different vertical sizes on mobile based on page

Change Summary:
---------------
* Updated Schedule Props
* Updated Schedule SCSS

Dev-Ops
=======
If you added an ENV variable, please check off that you've updated the following  
- [ ] Key & Sample value to `.env` & `sample.env` 
- [ ] GCP Secret Manager (Both development and production)
- [ ] Github Secrets (Added a development and production variable)
- [ ] Github Workflows `.github/workflows/dev.yml` & `.github/workflows/prod.yml`
- [ ] webpack-config.js

Images/Important Notes (optional):
-----------------------
* Originally the component was designed to be the same size on mobile screens but due to how empty the component is, this was changed
# Desktop
updated-schedule-desktop-d1
updated-schedule-desktop-d2
updated-schedule-desktop-d3

# Mobile
updated-schedule-mobile-d1
updated-schedule-mobile-d2
updated-schedule-mobile-d3

### CODE REVIEW

File: src/Props/schedule/schedule.ts [hunk 1]
```diff
@@ -18,13 +18,17 @@ export const schedule: ScheduleDate[] = [
       { name: "Opening Ceremony", time: "7:00 PM" },
       { name: "Hacking Begins", time: "9:00 PM" },
       { name: "Resume/Cover Letter", time: "10:00 PM" },
-      { name: "Equity In Tech", time: "10:30 PM" },
+      { name: "MLH Event: Bob Ross Painting", time: "" },
```
Comment (sidhantas) [inline src/Props/schedule/schedule.ts:21]: This is missing a time so it doesn't show up on the schedule

**Review by sidhantas — APPROVED** [review]
LGTM!

**Review by Alperdec — APPROVED** [review]
thanks! looks a lot better on mobile than it did before

**Review by w-gao — APPROVED** [review]
Looks good! Thanks @krishnakpandian :).

**Pull request merged**